samples/drivers/adc/src/main.c

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2020 Libre Solar Technologies GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <sys/printk.h>
+#include <drivers/adc.h>
+
+/*
+ * ADC devicetree nodes have different naming conventions depending on the MCU
+ * vendor.
+ */
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(adc), okay)
```

Comment:
Would probably be good to require an alias as some other samples do (e.g. blinky)

Reply:
Yeah, thought about that, but I was wondering if this sample is worth touching almost every board dts. But that would be the cleanest approach, so I agree.
Are there any preferences for the alias name? Just `adc`?